server/graphql-schema.js

```diff
@@ -565,11 +599,11 @@ const graphqlSchema = gql`
         """
         The AtVersion used during this testing session.
         """
-        atVersion: AtVersion!
+        atVersion: AtVersion
         """
         The BrowserVersion used during this testing session.
         """
-        browserVersion: BrowserVersion!
+        browserVersion: BrowserVersion
```

Comment:
This is related to this task which is [being tracked as a follow-up](https://app.asana.com/0/1203924324409025/1206070969892275/f). I was experiencing intermittent issues with the entire app breaking due to null in the `browserVersion` or `atVersion` of a test result. I couldn't validate this state in the DB. Since this is a schema type that is used for reading, I thought it safe to disable this hard requirement. Any bugs that arise from a possibly null read on this object are likely to be less severe than the full-stop break of a failed GraphQL query (though I was unable to locate instances where even these less severe bugs could happen).

Honestly, this one was so intermittent that it might not even be relevant anymore since I can add those exclamation marks back and pass tests/manual tests without any problems currently. It does feel safest to let the person who handled that debt task make that determination though and until then leave the schema in a state that is least likely to break the app.

Reply:
Ah ok cool, would it be possible to make a note either in the task or with a TODO in the code that says the expected resolution is to make this field required again?

Reply:
Good idea. Done!